tools/update_runnablejobs.py

```diff
@@ -47,12 +47,13 @@ def update_runnableapi():
     # The format of expires is like 2017-07-04T22:13:23.248Z and we only want 2017-07-04 part
     expires = latest_task['expires'].split('T')[0]
     time_tuple = datetime.datetime.strptime(expires, "%Y-%m-%d").timetuple()
-    new_timestamp = time.mktime(time_tuple)
+    new_timestamp = time.mktime(time_tuple) # e.g. 1499140800.0 seconds since the epoch
     path = get_rootdir() + '/runnablejobs.json'
 
     # we do nothing if the timestamp of runablejobs.json is equal with the latest task
     # otherwise we download and update it
     if os.path.isfile(path):
+        # XXX: this is inefficient as json loading such big file is a slow method
```

Comment:
is runnable jobs that large?  would there be a better way to support getting all the valid job types?